docs/application/native/tutorials/process/performance.md

```diff
@@ -18,10 +18,10 @@ To improve the performance of an application:
 
 The Tizen Studio supports various tools which help to measure and increase performance at runtime:
 
-- [Dynamic Analyzer](../../../tizen-studio/common-tools/dynamic-analyzer/overview.md)  
+- [Dynamic Analyzer](../../../tizen-studio/common-tools/dynamic-analyzer/overview.md)
```

Comment:
line 21,24,50,53,62,89,92,178,179,180,183: The separated lines are merged as one line.
line 314: The separated lines are merged as one line.(max snapshots)

Reply:
done